### ISSUE TITLE
Can't download 1.5 model

### ISSUE THREAD

**Issue by Dreason8** [TheLastBen/fast-stable-diffusion#193]
Getting the following error when running the colab:

Access denied with the following error:
Too many users have viewed or downloaded this file recently. Please
	try accessing the file again later. If the file you are trying to
	access is particularly large or is shared with many people, it may
	take up to 24 hours to be able to view or download the file. If you
	still can't access a file after 24 hours, contact your domain
	administrator. 

Guessing everyone is after that 1.5 update?

**Comment by TheLastBen** [comment]
I'll fix that shortly

**Comment by Dreason8** [comment]
Yep working now. Thanks for your help and all your hard work behind the scenes.

**Comment by TheLastBen** [comment]
happy to help
